src/rules/no-standalone-expect.ts

```diff
@@ -4,21 +4,26 @@ import {
 } from '@typescript-eslint/experimental-utils';
 import { createRule, isDescribe, isExpectCall, isFunction } from './tsUtils';
 
-const getBlockType = (stmt: TSESTree.BlockStatement): String | false => {
+const getBlockType = (
+  stmt: TSESTree.BlockStatement,
+): 'function' | 'describe' | 'callExpr' | false => {
   const func = stmt.parent;
+
+  if (!func) {
```

Comment:
@SimenB So it looks like this is the causing the test coverage to not be 100%, do you know of any way to test this? when would the BlockStatment not have a parent?

Reply:
You might be able to sneak around this by moving [line 19](https://github.com/jest-community/eslint-plugin-jest/pull/350/files/9656d908b7a2526ac01e52652948d2cb16b9f0c2..d24f4afeeb6955569081938ac5b5bf8364654829#diff-8a6284e5ebef9b04182b8b9972128c4bR19) up (you also might have to put the `func.parent` before the `isFunction` check).

Reply:
I moved it out explicitly - if we can't cover that condition I think we should be explicit about it rather than combining conditions and "fooling" coverage

Reply:
Ah fair enough.

tbh I think I was thinking of methods anyway:

```
export const isExpectCall = (node: TSESTree.Node): node is ExpectCall =>
  node.type === AST_NODE_TYPES.CallExpression &&
  isExpectAccessor(node.callee) &&
  node.parent !== undefined;
```

I don't think `node.parent !== undefined` is ever not true, but coverage doesn't complain b/c it's not in an `if` branch.

> we should be explicit about it

Absolutely!